src/legacy/ui/public/vis/editors/default/agg_param.js

```diff
@@ -100,9 +109,14 @@ uiModules
             }
           };
 
+          $scope.setTouched = () => {
+            ngModelCtrl.$setTouched();
+            $scope.isInvalid = !$scope.paramValue;
```

Comment:
I don't think we want to generically set `isInvalid` here, based on the param, because this would now also happen for non required fields. I think that is only supposed to happen to required fields?